hack/swagger-doc/README.md

```diff
@@ -1,5 +1,5 @@
 Generating API documents
 ========================
 
-To generate docs, you must have Gradle installed at version 2.2+. Run `hack/gen-swagger-docs.sh` to create a new copy of
+To generate docs, it currently uses 'k8s.io/kubernetes/pkg/runtime'. Run `hack/update-generated-swagger-descriptions.sh`  to create a new copy of
```

Comment:
I don't think this is correct. Generating the ASCIIdoc documentation and the Swagger descriptions are different tasks. I believe @smarterclayton changed `hack/gen-swagger-docs.sh` to `hack/update-generated-docs.sh`.